packages/sveltekit/src/server/load.ts

```diff
@@ -59,7 +63,18 @@ export function wrapLoadWithSentry<T extends (...args: any) => any>(origLoad: T)
   return new Proxy(origLoad, {
     apply: (wrappingTarget, thisArg, args: Parameters<T>) => {
       // Type casting here because `T` cannot extend `Load` (see comment above function signature)
-      const event = args[0] as LoadEvent;
+      // Also, this event possibly already has a sentry wrapped flag attached
+      const event = args[0] as PatchedLoadEvent;
+
+      if (event.__sentry_wrapped__) {
+        return wrappingTarget.apply(thisArg, args);
+      }
+
+      const patchedEvent: PatchedLoadEvent = {
+        ...event,
+        __sentry_wrapped__: true,
+      };
```

Comment:
h: Can we use `addNonEnumerableProperty` and mutate the original event? This way we can just do `wrappingTarget.apply(thisArg, args)` like before, which is more safe if more args are added in the future.

```suggestion
      addNonEnumerableProperty(event, '__sentry_wrapped__', true);
```

Reply:
Yeah we can do that easily for the server-side, good call! On the client side, we have to patch the event for the fetch instrumentation.

Reply:
Done in [1d1dd92](https://github.com/getsentry/sentry-javascript/pull/8094/commits/1d1dd92e0d7a33d17d1bc991a9299566225e0cef)